src/System.Windows.Extensions/src/System/Drawing/FontConverter.cs

```diff
@@ -14,6 +14,8 @@ namespace System.Drawing
 {
     public class FontConverter : TypeConverter
     {
+        private const string StyleHdr = "style=";
```

Comment:
Nit, we generally avoid contractions in identifier names (well, str is fairly common, and id, and some single letters..) so normally we would write "StyleHeader" (or "StylePrefix" perhaps)